include/SoapySDR/Version.h

```diff
@@ -35,7 +35,7 @@
  * And <i>extra</i> is empty for releases but set on development branches.
  * The ABI should remain constant across patch releases of the library.
  */
-#define SOAPY_SDR_ABI_VERSION "0.8"
+#define SOAPY_SDR_ABI_VERSION "0.8-1"
```

Comment:
The internet people are going to come after me with pitchforks and torches. I didnt tag it yet. Just throw a line in the changelog. Its not really breaking anything. I'll merge this before the tag.

Reply:
I mean dont change the magic numbers.

Reply:
Done